project_template/inc/az/exampleshortname.h

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/** @file exampleshortname.h Azure example library */
+#include <az/exampleshortname/operations.h>

```

Comment:
>< [](start = 9, length = 1)

Chapter 6.10.2 Source file inclusion in C11 standard contains all the details about the difference between include< and include ".

In this case consider having include ".

Reply:
in this case would you want #include "exampleshortname/operations.h"?